recipes/repo.rb

```diff
@@ -111,35 +111,100 @@
   triggers_reload true
```

Comment:
This file appears to suffer from a pretty clumsy rebase. Please hold off on review of it for now.